src/client/dfuse/dfuse_pool.c

```diff
@@ -0,0 +1,151 @@
+/**
+ * (C) Copyright 2016-2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+#include "dfuse_common.h"
+#include "dfuse.h"
+#include "daos_fs.h"
+#include "daos_api.h"
+
+/* Lookup a container within a pool */
+static bool
+dfuse_pool_connect(fuse_req_t req, struct dfuse_inode_entry *parent,
+		   const char *name, bool create)
```

Comment:
Based on the name, one would expect this function connect to a pool whereas it actually open a container. That's a bit confusing IMO.

Reply:
Yes, I found it confusing as well, and hadn't even spotted the connect/open confusion.  I've tried a different tack and am renaming this entire file to dfuse_cont.c on the basis that it accesses containers, rather than being something you do from a pool inode.